webservices/resources/legal.py

```diff
@@ -145,7 +154,12 @@ def mur_query_builder(q, type_, from_hit, hits_returned, **kwargs):
         .index('docs_search') \
         .sort("sort1", "sort2")
 
-    return apply_mur_specific_query_params(query, **kwargs)
+    if type_ == 'murs':
```

Comment:
Instead of having distinctly named query parameters for every document type, we may want to consider combining them and having common parameters like document_no, respondents, dispositions, min_date, etc.